.github/workflows/ci.yml

```diff
@@ -51,11 +51,6 @@ jobs:
 
       - run: npm ci
 
-      - name: Check Cargo.lock
```

Comment:
This check failed on PR because `ruby-sdk` now relies on yet-unpublished `eppo_core@2.0.0`.

I have moved this check to `publish` workflow because we only need the lock file for published version